### PR TITLE
fix: videojs doesn't export individual items but an object

### PR DESCRIPTION
**Pull request by gkatsev**
Video.js exports an object and you can't do destructuring in imports, it's a special construct for named exports.
Also, modify the default export. Though, I can move that to a separate PR.

### CODE REVIEW

File: src/videojs-http-streaming.js [hunk 1]
```diff
@@ -698,7 +698,7 @@ if (videojs.registerPlugin) {
   videojs.plugin('reloadSourceOnError', reloadSourceOnError);
 }
 
-module.exports = {
+export default {
```
Comment (gkatsev) [inline src/videojs-http-streaming.js]: this should be equivalent but can be moved to a separate PR.

File: src/playlist-loader.js [hunk 2]
```diff
@@ -6,10 +6,11 @@
  *
  */
 import resolveUrl from './resolve-url';
-import { mergeOptions, EventTarget, log } from 'video.js';
+import videojs from 'video.js';
 import { Parser as M3u8Parser } from 'm3u8-parser';
 import window from 'global/window';
 
+const { mergeOptions, EventTarget, log } = videojs;
```
Comment (forbesjo) [inline src/playlist-loader.js]: Needs a new line

File: src/videojs-http-streaming.js (hunk 1 above)
Comment (forbesjo) [reply src/videojs-http-streaming.js]: I think it's fine to have this change in this PR
